src/persistence/job.js

```diff
@@ -12,18 +12,58 @@ module.exports = {
       }, ${jobParameters.description}, current_date, ${jobParameters.tags}, ${
         jobParameters.company
       }, ${jobParameters.logoURL})
-        RETURNING id, title, tags;
+        RETURNING *;
       `);
       return rows[0];
     } catch (error) {
       throw error;
     }
   },
-
   async find(id) {
     const {rows} = await db.query(sql`
     SELECT * FROM jobs WHERE id=${id} LIMIT 1;
     `);
     return rows[0];
+  },
+  async update(isJobExist, jobParameters) {
```

Comment:
The name `isJobExist` is surprised me. I expect it as a boolean parameter. However, it turns out that it's a job object. It's weird to call `isJobExist.title`, right? Do you have a better name?

Reply:
"oldJob" make sense, isn't it?

Reply:
Make it simpler. It's a `job`, right? :D